test/parallel/test-http-outgoing-internal-headers.js

```diff
@@ -31,3 +31,14 @@ common.expectWarning('DeprecationWarning', warn, 'DEP0066');
       origin: ['Origin', 'localhost']
     }));
 }
+
+{
+  // Tests for _headers set method `null`
+  const outgoingMessage = new OutgoingMessage();
+  outgoingMessage._headers = null;
+
+  assert.strictEqual(
+    outgoingMessage[kOutHeaders],
+    null
+  )
```

Comment:
```suggestion
  );
```

Reply:
Is there an automatic formatting tool?

Reply:
@ycjcl868 Yes, it could be fixed automatically by `eslint --fix`. If you use VSC, https://github.com/Microsoft/vscode-eslint would help to fix es-lint problem when you save file.

Reply:
Could this be added to a pre-commit?

Reply:
@ycjcl868 

Personally, it's nice to have.. 👍